src/components/Authorization/components/Login/login.jsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import Button from "../../../../common/Button/button";
+import Input from "../../../../common/Input/input";
+
+export const Login = (props) => {
```

Comment:
As you are not using TS, you should use PropTypes to validate props

Reply:
updated proptypes